store/tikv/lock_resolver.go

```diff
@@ -342,7 +347,9 @@ func (lr *LockResolver) resolveLocks(bo *Backoffer, callerStartTS uint64, locks
 				cleanTxns[l.TxnID] = cleanRegions
 			}
 
-			if l.LockType == kvrpcpb.Op_PessimisticLock {
+			if status.secondaries != nil && len(status.secondaries) > 0 {
```

Comment:
If the async commit txn has only one key, it should also be resolved in the async commit way.